views/components/GraphTable/GraphTable.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+import { formatSiteData, graphTableColumns } from '../../fe-utils/helpers'
+import { graphTableRowDataBySiteUtil } from './helpers'
+
+const GraphTable = ({ graph }) => {
+  if (!graph) return null
+
+  const tableColumns = graphTableColumns(graph.labels)
+  const [totalsSite, ...sites] = graphTableRowDataBySiteUtil(graph.dataBySite)
+
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {tableColumns.map(({ name }) => (
+              <React.Fragment key={name + 'table-header'}>
+                <TableCell align="center">{name}</TableCell>
+              </React.Fragment>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {sites.map((site) => {
+            return (
+              <TableRow hover key={site.name}>
+                <TableCell align="center">{site.name}</TableCell>
+                {tableColumns.map((column) => {
+                  const { name: columnLabel, color } = column
+                  const {
+                    counts: { [columnLabel]: siteCount },
+                    targets: { [columnLabel]: siteTarget },
+                    percentages: { [columnLabel]: sitePercentage },
+                  } = site
+                  return (
+                    <React.Fragment key={columnLabel + 'site-row'}>
+                      <TableCell align="center" style={{ color }}>
+                        {formatSiteData(siteCount, siteTarget, sitePercentage)}
+                      </TableCell>
+                    </React.Fragment>
```

Comment:
No need for a fragment here.

```suggestion
                    <TableCell align="center" key={columnLabel} style={{ color }}>
                      {formatSiteData(siteCount, siteTarget, sitePercentage)}
                    </TableCell>
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/193/commits/325e6635a9f09c0b3ce28c6d2a5ab8bfa1369601